index.js

```diff
@@ -40,7 +40,7 @@ exports.link = opts => {
 			const filePath = result.path;
 
 			const src = path.dirname(filePath);
-			const dest = path.join(workflowDir, pkg.name);
+			const dest = path.join(workflowDir, pkg.name.replace('/', '-'));
```

Comment:
Is it possible to have multiple slashes? Probably not, just wondering. Otherwise we would have to do `.replace(/\//g, '-')`.

Reply:
No, only one slash is possible.